src/com/georgidinov/util/Validator.java

```diff
@@ -0,0 +1,12 @@
+package com.georgidinov.util;
+
+public final class Validator {
+
+    private Validator() {
+    }
+
+    public static boolean isValidString(String stringToValidate) {
```

Comment:
Class with 1-2 static validation methods is not useful, just implement in once class and import from there